front/app/modules/commercial/segment/index.tsx

```diff
@@ -96,7 +96,7 @@ const configuration: ModuleConfiguration = {
                 locale: user.data.attributes.locale,
                 isSuperAdmin: isSuperAdmin(user),
                 isAdmin: isAdmin(user),
-                isProjectModerator: isProjectModerator(user),
+                isProjectModerator: isModerator(user),
```

Comment:
This is the same check as using isProjectModerator without a projectId, which will allow us to make isProjectModerator stricter (by requiring projectId of type string).